infogami/core/code.py

```diff
@@ -351,26 +385,28 @@ def POST(self):
             add_flash_message('info', 'Password updated successfully.')
             raise web.seeother("/account/preferences")
 
+
 register_preferences(change_password)
 
+
 class getthings(delegate.page):
     """Lists all pages with name path/*"""
+
     def GET(self):
         i = web.input("type", property="key")
-        q = {
-            i.property + '~': i.q + '*',
-            'type': i.type,
-            'limit': int(i.limit)
-        }
+        q = {i.property + '~': i.q + '*', 'type': i.type, 'limit': int(i.limit)}
```

Comment:
This change seems less clear than the original. I'm surprised this is joining dicts like this.

Reply:
`black` wants to keep everything in one line unless that exceeds the character limit. In cases like this, we can force `black` to keep the entries on separate line by adding a trailing comma.

Should I add it?

Reply:
# fmt: off 
_code that I do not want reformatted_
# fmt: on

Reply:
Actually, it is a good practice to keep a trailing comma to have a sane git diff. There's an entire plugin just to add that comma and a flake8 plugin as well to lint files for that comma.
Formatting plugin: https://github.com/asottile/add-trailing-comma
flake8 plugin: https://github.com/PyCQA/flake8-commas

Reply:
The very first line of the psf/blacks README says:
> Black is the uncompromising Python code formatter. By using it, you agree to cede control over minutiae of hand-formatting. In return, Black gives you speed, determinism, and freedom from pycodestyle nagging about formatting. You will save time and mental energy for more important matters.

I tend to want to _cede control over minutiae_ but...

Reply:
At the end, it's only about consistency with whichever style we choose.